src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const keyName of action.keysToRemove) {
+          delete state[keyName];
```

Comment:
```suggestion
        for (const key of action.keysToRemove) {
          delete state[key];
```
так теж буде зрозуміло що це ім'я ключа

Reply:
Agree